src/notebooks/controllers/kernelSource/kernelSelector.ts

```diff
@@ -77,13 +95,39 @@ export class KernelSelector implements IDisposable {
         this.displayDataProvider =
             ServiceContainer.instance.get<ConnectionDisplayDataProvider>(ConnectionDisplayDataProvider);
         this.extensionChecker = ServiceContainer.instance.get<IPythonExtensionChecker>(IPythonExtensionChecker);
+        this.createPythonEnvQuickPickItem = {
+            label: `$(add) ${DataScience.createPythonEnvironmentInQuickPick()}`,
+            command: this.onCreatePythonEnvironment.bind(this)
+        };
+        this.installPythonItem = {
+            label: DataScience.installPythonTitle(),
+            command: async () => {
+                // Timeout as we want the quick pick to close before we start this process.
+                setTimeout(() => commands.executeCommand(Commands.InstallPythonViaKernelPicker).then(noop, noop));
+                throw new SomeOtherActionError();
+            }
+        };
+        this.installPythonExtension = {
```

Comment:
Support:
* Install Python extension
* Install Python runtime
In the new kernel picker

These are commands displayed in the old (currently stable) version of the kernel picker ui.
We need to bring them into the new kernel picker as well.
All we're doing here is adding the items into the quick pick